SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.cxx

```diff
@@ -233,6 +233,7 @@ const TMap<Worker_ComponentId, FString> ServerAuthorityWellKnownComponents = {
     { SERVER_ENDPOINT_COMPONENT_ID, "unreal.generated.UnrealServerEndpoint" },
     { CROSSSERVER_SENDER_ENDPOINT_COMPONENT_ID, "unreal.generated.UnrealCrossServerSenderRPCs" },
     { CROSSSERVER_RECEIVER_ACK_ENDPOINT_COMPONENT_ID, "unreal.generated.UnrealCrossServerReceiverACKRPCs" },
+	{ MIGRATION_DIAGNOSTIC_COMPONENT_ID, "unreal.MigrationDiagnostic" },
```

Comment:
Super nit, but we have a tab here, while the other ones are 4 spaces, I think. Though not sure why the other ones are 4 spaces, since our usual style guide says tabs...